packages/arb-avm-cpp/app/main.cpp

```diff
@@ -119,7 +119,7 @@ int main(int argc, char* argv[]) {
               << assertion.gasCount << " gas ending in state "
               << static_cast<int>(mach->currentStatus()) << "\n";
 
-    auto tx = storage.makeReadWriteTransaction();
+    auto tx = storage.getReadWriteTransaction();
```

Comment:
What's the logic behind the name change? Seems like we're making a new one, not getting an existing one

Reply:
👍